java/src/org/openqa/selenium/chrome/ChromeDriver.java

```diff
@@ -91,7 +96,11 @@ public static RemoteWebDriverBuilder builder() {
 
   private static class ChromeDriverCommandExecutor extends ChromiumDriverCommandExecutor {
     public ChromeDriverCommandExecutor(DriverService service) {
-      super(service, getExtraCommands());
+      this(service, ClientConfig.ClientConfig.defaultConfig());
```

Comment:
Is this correct? Not sure if it compiles. Probably one `ClientConfig.` less?

Please be sure to do something like `bazel build grid` locally to verify the code compiles.